components/brave_rewards/resources/ui/components/tableAdsHistory/style.ts

```diff
@@ -55,7 +55,7 @@ export const StyledAdLink = styled('a')<{}>`
   text-decoration: none;
   cursor: pointer;
   display: flex;
-  color: ${p => p.theme.color.text};
+  color: ${p => p.theme.color.text} !important;
```

Comment:
This is needed to work around the where function introduced here 9e5cdb8cf34952261d47a3ff14167b870f903de3.

Reply:
FYI the issue here is that both this ruleset and the "default link color" ruleset have the same specificity (using 1 css class name).

Since this is in legacy UI code, the `!important` is OK with me. If this were in new code, we would want to avoid `!important` by increasing the selector specificity. (It probably wouldn't arise since we use nested `a {}` rulesets).